### DISCUSSION TITLE
Cannot start server

### DISCUSSION THREAD

**Discussion by T-449** [open-quantum-safe/oqs-provider#452] [Q&A]
hi,
I was looking top start the server using the commands specified in the readme. but it gives the following error
![Screenshot from 2024-07-22 16-27-00](https://github.com/user-attachments/assets/3439fcc7-914d-4492-9b5d-5633ede1e200).

**Comment by baentsch** [comment]
Did you read https://github.com/open-quantum-safe/oqs-provider/blob/main/USAGE.md#activation ? Or consider using our ready-made dockerimages, e.g., https://hub.docker.com/repository/docker/openquantumsafe/oqs-ossl3/general
